app/services/lambda_jobs/runner.rb

```diff
@@ -7,7 +9,7 @@ def initialize(job_class:, args:)
       @args = args
     end
 
-    def run(&local_callback)
+    def run()
```

Comment:
If we were going to ditch the block (which I'm not sure we're ready for yet), we should drop the parens too

```suggestion
    def run
```